<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hue</class>
 <widget class="QWidget" name="Hue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HueLights</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>-24</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Day Scenes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add Scene</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <row>
         <property name="text">
          <string>Night</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Morning</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Day</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Evening</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Scene</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Scene State</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Scene time start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Delete Scene</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>Night</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>Morning</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>19%</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>Day</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>81%</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>Evening</string>
         </property>
        </item>
        <item row="3" column="1">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
